packages/fluentui/docs/src/config.ts

```diff
@@ -1,4 +1,5 @@
 const config = {
+  repoURL: 'https://github.com/microsoft/fluentui',
```

Comment:
Should we import it from `packages/fluentui/react-northstar/src/utils/constants.ts` ? So we don't have to change URL in two places

Reply:
`constants` are no longer reexported under `@fluentui/react-northstar` (I think that this variable was exported by a mistake).